qa/tasks/deepsea_deploy.py

```diff
@@ -589,6 +611,12 @@ def _run_command_dict(self, cmd_dict):
         elif directive == "stage3":
             config = cmd_dict['stage3']
             target = self._run_stage_3
+        elif directive == "stage4":
+            config = cmd_dict['stage4']
+            target = self._run_stage_4
+        elif directive == "state_orch":
+            config = cmd_dict['state_orch']
+            target = self._state_orch
```

Comment:
I don't know if you like it but, what do you think about using the code:
```
method_name = "_run_" + directive
if (hasattr(self.__class__, method_name)):
    config = cmd_dict[directive]
    target = getattr(self.__class__, method_name)
else:
    raise ConfigError("deepsea_deploy: unknown directive '%s'" % directive)
```
and you'll need to rename methods correspondingly to *_run_stageN*, *_run_state_orch* and so on...

Reply:
Actually, I'm thinking about making a separate class for the directives...